RecoTauTag/TauTagTools/plugins/TFormulaWriter.cc

```diff
@@ -0,0 +1,66 @@
+#include "RecoTauTag/TauTagTools/plugins/TFormulaWriter.h"
```

Comment:
please move all non-production code to /test directory.
I don't see anything tau reco specific here.
This code should be in CondFormats/PhysicsToolsObjects/test or in a similar package.